How to Find and Use Technical Resources if You’re New to Tech:proposal.md

```diff
@@ -0,0 +1,87 @@
+
+## Video Tutorials and Walkthroughts  
+
+Earlier, I said that search engines like Google are your best friends, but that's not 100% accurate. In reality, Google AND Youtube are your best friends. You can look for video tutorials and walkthroughts of all levels where an expert can *show* you how to code and give you an explanation. 
+
+![](https://i.imgur.com/E3zzaD6.jpg)
+
+Devrel developer Kathy Liu likes using Youtube tutorials about the theory of programming. She said, "From the tutorials, you can learn the concepts and the bigger view of that, and then you can apply them to your code."
+
+Another way to use Youtube videos is to look up tutorials on specific coding projects. Jayaram said that he likes to "code along [with these types of tutorials] and add some new features" to make it more unique and better suit his goals as a programmer.  
+
+## Online Coding Lessons and Assessments
+
+If you are looking to learn with more structure, there are plenty of sites dedicated to teaching coders. 
+
+On one hand, a place to start if you're a novice programmer is freeCodeCamp. This nonprofit organization teaches users code through easy-to-manage lessons. They also have Youtube videos and can connect you to a community of others who are just as excited about coding as you are.
+
+![](https://i.imgur.com/lvwpLIA.png)
+
+On the other hand, there are also sites for you if you're more advanced and want more rigorous assessments to practice your skills (perhaps even to prepare you for technical interviews for programming jobs!). Kyle Begovich, director of developer relations, suggests [HackerRank](https://www.hackerrank.com/) or [LeetCode](https://leetcode.com/). These are great resources that can help teach you common practices in the industry and how to approach them. 
+
+## A Positive Mindset to Coding
+
+Now that you know some helpful resources, you can learn as little or as much code as you want. All you have to do is utilize the hundreds of free websites, videos, and documentation that are already out there on the internet. 
+
+However with so much information out there, it's understandable that you as a programmer might feel a bit overwhelmed. Perhaps you might even feel like there's so much to learn that you'll never be good enough as others. 
+
+When you've hit that sort of wall — a mental obstacle, not just a technical bug hindering your code — it can be difficult to find a way to over come it.  
+
+A bit of advice? Begovich says, "The imposter syndrome is a mistake people [who want to code] make. You come to the conclusion that you don’t belong here, [you] don’t have any original ideas. But if you have figured out this idea is done and this idea done, you can start to figure out that next step. That's where the learning takes place."
```

Comment:
Keep top 2-3 quotes and highlight them, integrate everything else into the blog as part of your research. For most developers/in our experience/personally/from interviewing Bit Project's members/according to these people/.....

Reply:
There's 20 " quotation marks on this page - you need to cut it down to 6.